java/code/src/com/redhat/rhn/manager/formula/FormulaManager.java

```diff
@@ -348,6 +348,15 @@ public void enableFormula(String minionId, String formulaName) throws IOExceptio
                 .collect(Collectors.toList());
     }
 
+    public List<Map<String, FormulaData>> getCombinedFormulaDataForSystems(User user, List<Integer> systemIDs) {
```

Comment:
Would it make sense to pass a flag to this method to optionally get only the monitoring formulas?

Reply:
I also don't like the fact that we dump a lot of not needed and possibly sensitive data. In the second commit I've proposed an alternative new API method to get only relevant information about endpoints.